src/rollup/utils/persist/read.js

```diff
@@ -0,0 +1,18 @@
+import expiringLocalStorage from './expiring-local-storage';
+import readCookie from '../cookies/read-cookie';
+import deleteCookie from '../cookies/delete-cookie';
+
+export default function read (key) {
+  var item = expiringLocalStorage.getItem(key);
+
+  if (item == null) {
+    item = readCookie(key);
+
+    if (item != null) {
+      deleteCookie(key);
+      expiringLocalStorage.setItem(key, item);
```

Comment:
i suppose this should get set unconditionally, so the expiration date gets extended on read

Reply:
definitely

Reply:
done... just, not here. the change was made to the interface `expiringLocalStorage.getItem`. I added a test case for this...